tests/unit/test_charm.py

```diff
@@ -1,7 +1,251 @@
-# Copyright 2023 Canonical
+# Copyright 2023 Canonical Ltd.
```

Comment:
Not sure how strongly I feel about it, but in our team so far we've been keeping scenario tests clearly split from harness 'unit' tests.
We have them all in their own /tests/scenario folder. Especially if you plan to also use harness, it might be a good idea to keep them separated.

Reply:
In this new charm, I was explicitly trying out Scenario as a replacement for harness. I am not expecting any uses of harness.